app/views/licenses/_form.html.erb

```diff
@@ -0,0 +1,22 @@
+<% if @current_license.present? %>
+  <collapsible-section initially-expanded="false"
```

Comment:
@oliverguenther On my machine the `collapsible-section` directive does not show any effect